### ISSUE TITLE
POS Tagger Training: Algorithms

### ISSUE THREAD

**Issue by dulocian** [emorynlp/nlp4j#29]
Hi,

I had some trouble trying to train POS tagger models on my own data sets using one of the 5 other algorithms.

The [quickstart](https://emorynlp.github.io/nlp4j/quickstart/train.html) guide contains the following options under **optimizer**:
**algorithm**: perceptron, softmax, adagrad, agagrad-mini-batch, agadelta-mini-batch, agagrad-regression.

When editing the config file the following of those options are incorrect and need to be replaced with:
*  softmax  -  softmax-regression 
*  agadelta-mini-batch  -  a**d**adelta-mini-batch
*  agagrad-mini-batch  -  a**d**agrad-mini-batch
*  agagrad-regression  -  a**d**agrad-regression

I also noticed that **ffnn-softmax** is a possible option listed in the source code although not specified in the quickstart guide - when testing it it fails reporting a null pointer exception - will this option be available in future or is this a possible error?

**Comment by jdchoi77** [comment]
FFNN was developed for our internal experiments and it is functional but we didn't make it publicly available due to a lack of wrapper APIs. Thanks for those correction!

best,

Jinho
